examples/script/responses/1_basics.py

```diff
@@ -68,20 +66,17 @@ class CallbackRequest(NamedTuple):
 
 
 def process_request(ctx: Context):
```

Comment:
Repeated definition. Maybe define once in testing_utils?

Reply:
I think, we should keep it, maybe in future with best practices, but now it seems good examples of using